subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -67,235 +67,134 @@
                 <main class="main-content">
                     <nav class="docs-navigation">
                         <ul>
-                            <li><a href="/userguide/userguide.html">User Manual Home</a></li>
-                            <li><a href="/release-notes.html">Release Notes</a></li>
+                            <li>
+                                <a href="/userguide/userguide.html">User manual</a>
```

Comment:
The same as "User Manual Home". I think it's more useful as a link from the reference and tutorial guides than anything else. As we discussed, this may become the "Docs Home" at a later date, but for now I don't think the "Home" adds anything.

That said, I don't have a strong opinion on this.

Reply:
Also, the Guides currently have just "User Manual" in their nav, not "User Manual Home". [Example](https://guides.gradle.org/designing-gradle-plugins/).

Reply:
One option could be to make it _not_ a link for user manual chapters. I don't think changing this is critical though. Plus I don't see many (if any) folks wanting to get to the home page from elsewhere in the user manual since the same nav applies to all of them.

Reply:
👍